tronbyt_server/templates/manager/update.html

```diff
@@ -1,101 +1,214 @@
 {% extends 'base.html' %}
 {% block header %}
 <script src="{{ url_for('static', filename='js/location.js') }}"></script>
-<h1>{% block title %}{{ _('Edit') }} "{{ device['name'] }}"{% endblock %}</h1>
+<h1 class="page-title">{% block title %}{{ _('Edit Device') }}: "{{ device['name'] }}"{% endblock %}</h1>
+<style>
+  body {
```

Comment:
@tavdog these styles seem to work better with dark mode than with light mode:

<img width="1300" alt="Bildschirmfoto 2025-04-11 um 07 06 22" src="https://github.com/user-attachments/assets/5fdff635-df9d-445d-9225-d29a436d05e6" />

Should we have `@media` selectors here and/or move the styles to `styles.css` so that they can be reused in other forms?